frontend/src/component/project/Project/ProjectStatusModal.tsx

```diff
@@ -0,0 +1,21 @@
+import { styled } from '@mui/material';
+import { SidebarModal } from 'component/common/SidebarModal/SidebarModal';
+
+const ModalContentContainer = styled('div')(({ theme }) => ({
+    minHeight: '100vh',
+    display: 'flex',
+    backgroundColor: theme.palette.background.default,
+}));
+
+type Props = {
+    open: boolean;
+    close: () => void;
+};
+
+export const ProjectStatusModal = ({ open, close }: Props) => {
+    return (
+        <SidebarModal open={open} onClose={close} label='Import toggles'>
```

Comment:
Not sure about the label

Reply:
While you are at it, lets rename the original to flags.

Reply:
Oh, right. That's a copy-paste error. yeah, I'll fix